examples/restaurant_demo/restaurant.py

```diff
@@ -40,7 +40,7 @@ class RestaurantStatistics:
     def __init__(self, hours_filename, days_filename, epsilon=LN_3):
```

Comment:
In this constructor/`__init__` function, we should explain via comment what each of the variables does.

e.g.
```python
...
# Epsilon—the privacy threshold—a number between 0 and 1
self.epsilon = epsilon
```